### PR TITLE
Add tuned roadrunner, updates to TeleOP

### PR DESCRIPTION
**Pull request by osun24**
**Roadrunner:** Tuned for localization, feedforward controller, and follower PID controller - we are ready to begin testing auto paths with roadrunner (see videos on Teams). 

**Minor changes to TeleOP:** removed the wrist and fixed PIDF control in the joint along with enums tracking joint (shoulder) & arm (forearm) states - in collaboration with @j1won31 and @ShasshU. We will also need to add PIDF control to the arm.

### CODE REVIEW

**Review by j1won31 — APPROVED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Auto/PixelDropoffBlueDR.java [hunk 1]
```diff
@@ -149,9 +151,18 @@ public void runOpMode() throws InterruptedException {
             setArmPower(ARM_SCORE);
             stopArticulation();
             setClawServoLeft(ClawServoLeft, CLAW_LEFT_CLOSED);
+            sleep(50);
             setArmPower(0);
```
Comment (j1won31) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Auto/PixelDropoffBlueDR.java:155]: Why did you set ArmPower to 0? Doesn't this mean arm won't work?

Comment (j1won31) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Auto/PixelDropoffBlueDR.java:155]: Why did you set ArmPower to 0? Doesn't this mean arm won't work?

Comment (j1won31) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Auto/PixelDropoffBlueDR.java:155]: On line 152

**Review by j1won31 — APPROVED** [review]

**Review by j1won31 — APPROVED** [review]

**Review by j1won31 — APPROVED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java [hunk 2]
```diff
@@ -99,10 +100,10 @@ public SampleMecanumDrive(HardwareMap hardwareMap) {
                 DriveConstants.LOGO_FACING_DIR, DriveConstants.USB_FACING_DIR));
         imu.initialize(parameters);
 
-        leftFront = hardwareMap.get(DcMotorEx.class, "leftFront");
-        leftRear = hardwareMap.get(DcMotorEx.class, "leftRear");
-        rightRear = hardwareMap.get(DcMotorEx.class, "rightRear");
-        rightFront = hardwareMap.get(DcMotorEx.class, "rightFront");
+        leftFront = hardwareMap.get(DcMotorEx.class, "frontLeft");
+        leftRear = hardwareMap.get(DcMotorEx.class, "backLeft");
+        rightRear = hardwareMap.get(DcMotorEx.class, "frontRight");
```
Comment (j1won31) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java]: Isn't rightRear going to be backRight?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java [hunk 3]
```diff
@@ -99,10 +100,10 @@ public SampleMecanumDrive(HardwareMap hardwareMap) {
...
+        rightFront = hardwareMap.get(DcMotorEx.class, "backRight");
```
Comment (j1won31) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java]: Isn't rightFront going to be frontRight?

**Review by j1won31 — APPROVED** [review]

**Review by j1won31 — APPROVED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java (hunk 3 above)
Comment (j1won31) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java]: Never mindd I just saw the changes after this

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java (hunk 2 above)
Comment (j1won31) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java]: Never mindd I just saw the changes after this

**Review by j1won31 — APPROVED** [review]

**Review by j1won31 — APPROVED** [review]

**Review by j1won31 — DISMISSED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java (hunk 2 above)
Comment (osun24) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/SampleMecanumDrive.java]: good catch - I messed this up initially and found out while tuning

**Review by Paradoxicaly — APPROVED** [review]

**Review by Paradoxicaly — DISMISSED** [review]

**Comment by osun24** [comment]
@j1won31  @Paradoxicaly can you guys reapprove the PR? I kind of forgot that this wasn't merged and made changes to the branch

**Review by Paradoxicaly — APPROVED** [review]

**Review by j1won31 — APPROVED** [review]

**Pull request merged**